components/hab/src/cli.rs

```diff
@@ -905,6 +906,36 @@ fn sub_pkg_build() -> App<'static, 'static> {
     sub
 }
 
+fn sub_pkg_download(feature_flags: FeatureFlag) -> App<'static, 'static> {
+    let mut sub = clap_app!(@subcommand download =>
+    (about: "Download Habitat artifacts (including dependencies and keys) from Builder")
+    (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for Builder")
+    (@arg BLDR_URL: --url -u +takes_value {valid_url} default_value(habitat_core::url::DEFAULT_BLDR_URL)
+        "Specify an alternate Builder endpoint. If not specified, the value will \
+         be taken from the HAB_BLDR_URL environment variable if defined. (default: \
+         https://bldr.habitat.sh)")
+    (@arg CACHE_DIRECTORY: --cache +takes_value "The path to store downloaded artifacts")
```

Comment:
Maybe call this DOWNLOAD_DIRECTORY and default to the user's cache directory if it is not specified.

Reply:
I like that. The name seems clearer to me.